packages/asset-swapper/src/utils/quote_simulation.ts

```diff
@@ -172,7 +172,7 @@ export function fillQuoteOrders(
                     fo.totalOrderInput,
                     fo.totalOrderInputFee,
                 );
-                const filledOutput = subFill.output.times(filledInput.div(subFill.input));
+                const filledOutput = subFill.output.times(filledInput.div(BigNumber.max(1, subFill.input)));
```

Comment:
Fixes the odd NaN error when quantities are very small. There's still another one out there I'm trying to track down.